### DISCUSSION TITLE
PCL社区站？

### DISCUSSION THREAD

**Discussion by MoYuan-CN** [Meloong-Git/PCL#2926] [提问] [answered]
今天看帮助库交pr的时候忽然发现了个 https://pcl2-help.gitee.io
话说 这玩意还有人在运营么

**Comment by MoYuan-CN** [comment]
https://pcl2-help.gitee.io/post/b468abc5.html?t=1701363150099
从这下面过去的链接 Gitee 吐了个 403 回来

**Comment by WForst-Breeze** [comment, marked as answer]
停更了，源仓库地址如此：https://gitee.com/pcl2-help/pcl2-help

但是个人认为还是有保留的必要的，其涵盖了一些帮助库无法涵盖到的游戏过程中的细节问题

**Comment by allMagicNB** [comment]
> 今天看帮助库交pr的时候忽然发现了个 https://pcl2-help.gitee.io
> 话说 这玩意还有人在运营么

没人了，但是具有纪念意义（不是
